utils/fs.go

```diff
@@ -31,3 +32,12 @@ func WriteJSON(fs afero.Fs, dir, fileName string, data interface{}) error {
 	}
 	return nil
 }
+
+func JSONMarshal(t interface{}) ([]byte, error) {
+	buffer := &bytes.Buffer{}
+	encoder := json.NewEncoder(buffer)
+	encoder.SetEscapeHTML(false)
+	encoder.SetIndent("", "  ")
+	err := encoder.Encode(t)
+	return buffer.Bytes(), err
+}
```

Comment:
Why do we need this?

Reply:
This is to parse CPE. We have '<'  and '>' symbols in CPEMatch